src/scripts/sidebar.js

```diff
@@ -131,6 +132,11 @@ class SideBar extends EventEmitter {
         img.src = `${hostUrl}/assets/favicon.svg?v=${Math.round(Math.random()*10000)}`;
     }
 
+    fixTitleBar () {
+        const fixScript = 'document.body.innerHTML = document.body.innerHTML + "<style>.side-nav{  margin-top: 15px; }</style>"';
+        document.getElementsByTagName('webview')[0].executeJavaScript(fixScript);
```

Comment:
This will always add the margin even if you show the side bar. It will also only work on the first server.

Try moving the CSS injection to the `show` and `hide` functions like this in `hide`:
```
document.querySelectorAll('webview').forEach((webviewObj) => webviewObj.insertCSS('aside.side-nav{margin-top:15px;overflow:hidden;}'));
```
And this in `show`:
```
document.querySelectorAll('webview').forEach((webviewObj) => webviewObj.insertCSS('aside.side-nav{margin-top:0;overflow:auto;}'));
```

You could also create an injectCSS function in the webview class to reduce this duplicated code